app/create/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var Angular    = require('angular');
+var Controller = require('./controller');
```

Comment:
nit: new line before this line to separate our module requires from our relative requires. then you should realign the `=`s per block. here's a [reference](https://github.com/lob/wiki/blob/master/engineering/style-guides/node.md#group-internalexternal-dependencies) that explains it a bit better.